src/main/java/com/hillel/items_exchange/service/ImageService.java

```diff
@@ -3,23 +3,28 @@
 import com.hillel.items_exchange.dao.ImageRepository;
 import com.hillel.items_exchange.dto.ImageDto;
 import com.hillel.items_exchange.model.Image;
+import com.hillel.items_exchange.model.Product;
 import lombok.RequiredArgsConstructor;
 import org.modelmapper.ModelMapper;
 import org.modelmapper.TypeToken;
 import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
 
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 @Service
 @RequiredArgsConstructor
 public class ImageService {
     private final ModelMapper modelMapper;
+    private final ProductService productService;
     private final ImageRepository imageRepository;
 
-    public List<String> getLinksByProductId(Long id) {
+    public List<byte[]> getImageResources(Long id) {
```

Comment:
It would be nice to add to method name "ByProductId", because a little confusing